ksqldb-common/src/main/java/io/confluent/ksql/util/KsqlConfig.java

```diff
@@ -309,12 +309,6 @@
   public static final String KSQL_CREATE_OR_REPLACE_ENABLED_DOC =
       "Feature flag for CREATE OR REPLACE";
 
-  public static final String KSQL_ENABLE_METASTORE_BACKUP = "ksql.enable.metastore.backup";
```

Comment:
I'm not sure if we should remove this config. The idea behind having two configs, one to enable/disable and one to specify the location, was to allow users to disable backups temporary without removing the location. This to ensure users won't incorrectly type the location. KSQL creates the directory if the location doesn't exist.

Also, initially I thought of specifying a default backup location, so users would only enable/disable. If we can do this by setting the default to the streams directory, then having the on/off config would be very beneficial.

Reply:
They could just comment out the the backup location config to disable temporarily though right?